routes/error-tracker.js

```diff
@@ -54,6 +54,12 @@ async function logEvent(log, event) {
 async function buildEvent(req, reportingParams, logTarget) {
   const { buildQueryString, message, stacktrace, version } = reportingParams;
 
+  const userAgent = req.get('User-Agent');
+  if (userAgent.includes('Googlebot')) {
+    console.warn(`Ignored Googlebot errror report`);
```

Comment:
Can we log the error here. It won't go into the Errors interface, but at least we can look at the logs.

Reply:
Included the message here (not the full stacktrace) as I expect it should be enough to find a matching error in the interface